### PR TITLE
ipsec demo

### PR DESCRIPTION
**Pull request by mgheorghe**
- includes the files from networking api demo
- includes the files for the ipsec demo
- 
Signed-off-by: Mircea Dan Gheorghe mircea-dan.gheorghe@keysight.com

### CODE REVIEW

**Review by glimchb — APPROVED** [review]
Great!!

File: demos/security/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- enable ip forwarding `sysctl net.ipv4.ip_forward=1`
+- set ip addresses on the phisical links
+  ```
+  ip addr add 200.0.0.1/24 dev enp3s0f0s0
+  ip addr add 201.0.0.1/24 dev enp3s0f1s0
+  ```
+- install Docker (see [Docker manual](https://docs.docker.com/engine/install/ubuntu/) )
+- start OPI server
+```
+git clone https://github.com/opiproject/opi-strongswan-bridge
```
Comment (glimchb) [inline demos/security/README.md:33]: seems like you don;'t need/use clone anywhere...

File: demos/security/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    -d ghcr.io/opiproject/opi-security-server:main /opi-vici-bridge -port=50151
+
+
+
+# debug
+docker run --rm --network host \
+    --mount src=/var/run,target=/var/run,type=bind \
+    -it ghcr.io/opiproject/opi-security-server:main sh
+
+https://github.com/opiproject/opi-strongswan-bridge/blob/main/conf/strongswan-server.conf#L21
```
Comment (seroyer) [inline demos/security/README.md:51]: Should this be a comment?

File: demos/security/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+socket = unix:///var/run/charon.vici
```
Comment (seroyer) [inline demos/security/README.md:53]: What is one supposed to do with this?

**Review by seroyer — COMMENTED** [review]
I like it.  I have a couple of minor comments, but once you fix the linter errors we can merge it.

File: demos/security/README.md (hunk 1 above)
Comment (mgheorghe) [reply demos/security/README.md]: i removed it i, went that path till i noticed they were already built so there was no need to build myself teh python lib

File: demos/security/README.md (hunk 2 above)
Comment (mgheorghe) [reply demos/security/README.md]: i removed it i, went that path till i noticed they were already built so there was no need to build myself teh python lib

File: demos/security/README.md (hunk 3 above)
Comment (mgheorghe) [reply demos/security/README.md]: leftover from my notes, i removed it

**Pull request merged**